tests/e2e/metrics/metrics_test.go

```diff
@@ -231,10 +231,10 @@ func findHTTPMetricFromPrometheus(t *testing.T, app string, res *http.Response)
 					case "method":
 						if count.GetValue() > 0 {
 							switch val {
-							case "Healthz":
```

Comment:
Is there a way to have the information on the healthz calls too? Maybe add it to the path since it can be interpreted as an allowed path.

Reply:
I was holding that for another PR, adding well-known paths like `/dapr/config`, `/healthz`, etc

When we make that change it makes sense to add a new switch case testing the healthz path